airflow/contrib/example_dags/example_gcp_bigquery_dts.py

```diff
@@ -0,0 +1,89 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""
+Example Airflow DAG that creates and deletes Bigquery data transfer configurations.
+"""
+import os
+
+from google.protobuf.struct_pb2 import Struct, Value
+
+import airflow
+from airflow import models
+from airflow.contrib.operators.gcp_bigquery_dts_operator import (
+    BigQueryCreateDataTransferOperator,
+    BigQueryDeleteDataTransferConfigOperator,
+)
+from airflow.operators.bash_operator import BashOperator
+
+# [START howto_bigquery_dts_os_args]
+GCP_PROJECT_ID = os.environ.get('GCP_PROJECT_ID', 'example-project')
+# [END howto_bigquery_dts_os_args]
+
+# [START howto_bigquery_dts_create_args]
+PARAMS = Struct(
```

Comment:
It looks strange.
Did you contact Google about this matter?  Maybe this could be fixed in the google-cloud-python library. It could use a link to the issues on GH.

Reply:
We do not want to be just a worker, but a partner with a client. It requires communication about our and his expectations.  It builds a positive relationship with a client.